src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/PledgeService/PledgeService.cs

```diff
@@ -80,6 +80,14 @@ public async Task<GetApplicationsResponse> GetApplications(long accountId, int p
             return JsonConvert.DeserializeObject<GetApplicationsResponse>(await response.Content.ReadAsStringAsync());
         }
 
+        public async Task<GetApplicationsForDownloadResponse> GetApplicationsForCsvDownload(long accountId, int pledgeId)
+        {
+            var response = await _client.GetAsync($"accounts/{accountId}/pledges/{pledgeId}/applications/download");
```

Comment:
I might be wrong, but I thought we'd said we would simply extend the existing apim endpoint/response type, since the download link is "part of the page"